examples/jsm/node-editor/utils/PreviewEditor.js

```diff
@@ -1,8 +1,8 @@
-import { OrbitControls } from 'three-addons/controls/OrbitControls.js';
-import { ViewHelper } from 'three-addons/helpers/ViewHelper.js';
+import { OrbitControls } from '../../controls/OrbitControls.js';
```

Comment:
Why do you change here `three-addons` to `../..`?

Reply:
So we should add https://github.com/mrdoob/three.js/pull/23406 change of such `../..` in place of `three-addons` to `three/addons`.

Reply:
Perhaps. #23406 is a bigger change to the public API, and it sounds like people are still considering multiple npm packages, so I don't know what will happen there. But I don't think these couple of `three-addons` references are serving a necessary purpose in the meantime.

Reply:
Yeah let's keep these two PRs separate.

Reply:
No, I'm not suggesting to combine these PRs, I just think that if we are making such change in this PR we should also add a corresponding change to that one.

Reply:
Let's discuss there — not a big deal if https://github.com/mrdoob/three.js/pull/23406 misses one or two references to a `../../` prefix that could have been replaced with the alias though. 

Goal of this current PR is to eliminate use of incompatible aliases.